WordPress/UITestsFoundation/Screens/Login/Unified/PasswordScreen.swift

```diff
@@ -1,21 +1,14 @@
+import ScreenObject
 import XCTest
 
-private struct ElementStringIDs {
-    static let passwordTextField = "Password"
-    static let continueButton = "Continue Button"
-    static let errorLabel = "Password Error"
-}
-
-public class PasswordScreen: BaseScreen {
-    let passwordTextField: XCUIElement
-    let continueButton: XCUIElement
-
-    public init() {
-        let app = XCUIApplication()
-        passwordTextField = app.secureTextFields[ElementStringIDs.passwordTextField]
-        continueButton = app.buttons[ElementStringIDs.continueButton]
+public class PasswordScreen: ScreenObject {
 
-        super.init(element: passwordTextField)
+    public init(app: XCUIApplication = XCUIApplication()) throws {
+        try super.init(
+            // // swiftlint:disable:next opening_brace
```

Comment:
I'm not entirely sure how the double `//` will be handled by swiftlint, so just wanted to mention this:

```suggestion
            // swiftlint:disable:next opening_brace
```

Reply:
Oh, good catch, thanks!

I think the SwiftLint parser doesn't care about it, or we would have seen the warning here from Hound.